src/index.html

```diff
@@ -11,7 +11,184 @@
+
+        <label for="age">
+          How old are You?
+          <input 
+            id="age" 
+            class="age"
+            type="number" 
+            name="age" 
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label for="date">
+          Full date of birth:
+          <input 
+            id="date" 
+            type="date"
+            name="date" 
+            required
+          >
+        </label>
+
+        <label for="checkbox">
+          I accept the term of the agreement
+          <input 
+            id="checkbox" 
+            type="checkbox" 
+            name="agreement" 
+            required
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="regist">
+        <legend>Registration</legend>
+        <label for="email">
+          E-mail:
+          <input 
+            id="email" 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com" 
+            required 
+            multiple
+          >
+        </label>
+
+        <label for="password">
+          Password:
+          <input 
+            id="password" 
+            type="password" 
+            name="password" 
+            minlength="10" 
+            maxlength="20" 
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fact">
+        <legend>An interesting fact about you!</legend>
+        <label for="radio" class="cats">
+          Do you love cats?
+          <input 
+            id="radio"
+            type="radio"
+            name="radio"
+            value="y"
+          > 
+            Yes
+          <input 
+            id="radio1" 
+            type="radio" 
+            name="radio" 
+            value="n"
+          >
+            No
+        </label>
+        </div>
+
+        <label for="color">
+          What is your favorite color?
+          <input 
+            id="color" 
+            type="color"
+            name="color" 
+            required
+          >
+        </label>
+        
+        <label for="time">
+          What time do you go to bed?
+          <input 
+            id="time"
+            type="time" 
+            name="time" 
+            required
+          >
+        </label>
+
+        <label for="cars">
+          What are your favorite brand of cars?
+          <select 
+            id="cars" 
+            name="car-srlect" 
+            required
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label for="range">
+          How do you rate our work?
+          <input 
+            id="range" 
+            type="range" 
+            name="range"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="add">
+        <legend>Additional info:</legend>
```

Comment:
But there shouldn't be space between parent and child elements
```suggestion
            type="range" 
            name="range"
          >
        </label>
      </fieldset>
      
      <fieldset class="add">
        <legend>Additional info:</legend>
```